manifests/s/Synology/ChatClient/1.1.1-57/Synology.ChatClient.locale.en-US.yaml

```diff
@@ -0,0 +1,28 @@
+# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+
+PackageIdentifier: Synology.ChatClient
+PackageVersion: 1.1.1-57
+PackageLocale: en-US
+Publisher: Synology Inc.
+PublisherUrl: https://www.synology.com
+PublisherSupportUrl: https://www.synology.com/company/contact_us
+PrivacyUrl: https://www.synology.com/privacy
+PackageName: Synology Chat Client
```

Comment:
```suggestion
PackageName: Synology Chat
```

Reply:
I believe there's a mistake here; Synology Chat is the DSM package (for the NAS), whereas Synology Chat Client is the desktop software for it.

Reply:
https://global.download.synology.com/download/Utility/ChatClient/1.1.1-57/Windows/Chat%20Setup%201.1.1-57.exe
![image](https://user-images.githubusercontent.com/15158490/124394082-05150680-dcfe-11eb-8600-08f170366055.png)

Reply:
Alright let's put it like this then it's not a big deal. But it's confusing since the software is also referred as such in the official documentation.
https://kb.synology.com/en-ca/DSM/help/ChatClient/chatclient

Reply:
It does not matter, these Suggested Changes are REQUIRED for WinGet to work properly, if you have a problem with the Metadata the Software is reporting take it with the Developer of the Software, in this case Synology.

Reply:
I didn't know that, my bad.